requirements/edx/base.in

```diff
@@ -171,21 +150,48 @@ social-auth-core
 simplejson
 Shapely                             # Geometry library, used for image click regions in capa
 six                                 # Utilities for supporting Python 2 & 3 in the same codebase
-skill-tagging                       # Library needed to support tagging of skills in unit/video xblocks via forms
 social-auth-app-django
 sorl-thumbnail
 sortedcontainers                    # Provides SortedKeyList, used for lists of XBlock assets
 sqlparse                            # Required by Django to run migrations.RunSQL
 stevedore                           # Support for runtime plugins, used for XBlocks and edx-platform Django app plugins
-staff-graded-xblock                 # An XBlock that allows for off-site bulk scoring of problems
-super-csv                           # Generic CSV processor
 unicodecsv                          # Easier support for CSV files with unicode text
 user-util                           # Functionality for retiring users (GDPR compliance)
 webob
 web-fragments                       # Provides the ability to render fragments of web pages
 XBlock[django]                      # Courseware component architecture
 xblock-utils                        # Provides utilities used by the Discussion XBlock
 xss-utils                           # https://github.com/edx/edx-platform/pull/20633 Fix XSS via Translations
+
+########################################################################################################################
+## Non-kernel dependencies
+# The following dependencies do not seem to be core to the platform's functionality, and will soon be moved to a
+# separate "default.in" file. They may still be installed by default, but they should not be required for the platform
```

Comment:
Would you consider naming the new file `bundled.in` to match the [terminology in OEP-57](https://docs.openedx.org/projects/openedx-proposals/en/latest/processes/oep-0057-proc-core-product.html#bundled-extensions)? Along those same lines, we could eventually rename `base.in` to `kernel.in`.

Reply:
Yes, I like that a lot more. I just pushed a commit that split `base.in` into `kernel.in` and `bundled.in`.